code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -255,7 +255,7 @@
 	else if (!disk_rescued &&  crew_evacuated)
 		feedback_set_details("round_end_result","halfwin - detonation averted")
 		world << "<FONT size = 3><B>Syndicate Minor Victory!</B></FONT>"
-		world << "<B>[syndicate_name()] operatives recovered the abandoned authentication disk but detonation of [station_name()] was averted.</B> Next time, don't lose the disk!"
+		world << "<B>[syndicate_name()] operatives survived the assault but failed to secure the disk or detonate [station_name()].</B> Next time, don't lose the disk!"
```

Comment:
I've only seen people react that way a couple times, and both times several people answered the way RandomMarine just did